src/test/java/core/basesyntax/FileReaderServiceTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderServiceTest {
+    private static final String INPUT_FILE_EXIST = "src/main/resources/input.csv";
+    private static final String INPUT_FILE_NOT_EXIST = "src/main/resources/notExistFile.csv";

```

Comment:
In some moment you file can be removed. So, better if you add @BefforeClass (in 4.12 it have that name)) method, where you create test file`s.
And in @AfterClass you remove it